autoshop/core/forms.py

```diff
@@ -14,11 +15,27 @@ class SignupForm(forms.Form):
         password (str): Password of user
 
     """
-    first_name = forms.CharField(max_length=30)
-    last_name = forms.CharField(max_length=30)
-    email = forms.EmailField(max_length=100)
-    username = forms.CharField(max_length=30)
-    password = forms.CharField(widget=forms.PasswordInput, required=True)
+    GENDER_CHOICES = (
+        ('M', 'Male'),
+        ('F', 'Female'),
+    )
+    first_name = forms.CharField(help_text='Enter first name', max_length=30)
```

Comment:
if we are using `User` model as model form then do we need to declare/overwrite the fields again here?